Debug: Initializing model 'scratch' on device 'cpu'
Debug: Model 'ScratchModel' initialized
Debug: Model moved to device
Model 'ScratchModel' was created
Traceback (most recent call last):
  File "/Users/csl/GitHub_repository_bachelor/MDE02/pythonProject/train.py", line 191, in <module>
    main()
  File "/Users/csl/GitHub_repository_bachelor/MDE02/pythonProject/train.py", line 53, in main
    train_subset, val_subset = load_nyu_train(transform_image=transform_image, transform_depth=transform_depth, transform_pair=transform_pair)
  File "/Users/csl/GitHub_repository_bachelor/MDE02/pythonProject/data.py", line 90, in load_nyu_train
    train_data =   NYUDepth(
  File "/Users/csl/GitHub_repository_bachelor/MDE02/pythonProject/data.py", line 13, in __init__
    with open(csv_path, "r") as file:
FileNotFoundError: [Errno 2] No such file or directory: './data/nyudepthv2/train_files.csv'
